source/BrightcoveOS .NET-MAPI-Wrapper/Model/Items/BrightcoveCaptioning.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Web.Script.Serialization;
+using BrightcoveMapiWrapper.Serialization;
+using BrightcoveMapiWrapper.Util;
+
+namespace BrightcoveMapiWrapper.Model.Items
+{
+    /// <summary>
+    /// The Captioning object represents all closed captioning files and metadata assigned to a video.
+    /// </summary>
+    /// <remarks>
+    /// See https://docs.brightcove.com/en/video-cloud/media/references/reference.html#Captioning
+    /// </remarks>
+    public class BrightcoveCaptioning : BrightcoveItem, IJavaScriptConvertable
+    {
+        /// <summary>
+        /// A number that uniquely identifies this Captioning object, 
+        /// assigned by VideoCloud when this object is created.
+        /// </summary>
+        public long? Id { get; set; }
+
+        /// <summary>
+        /// A set of sources which provide caption. 
+        /// Only one CaptionSource is supported at this time.
+        /// </summary>
+        public ICollection<BrightcoveCaptionSource> CaptionSources { get; set; }
+
+        public IDictionary<string, object> Serialize(
+            JavaScriptSerializer serializer)
```

Comment:
That's quite a mish mash of different formatting styles. I guess we should stick to their (long oneliners?)